templates/shared/user/profile_big_avatar.tmpl

```diff
@@ -48,16 +48,8 @@
 				<li>
 					{{svg "octicon-mail"}}
 					<a class="tw-flex-1" href="mailto:{{.ContextUser.Email}}" rel="nofollow">{{.ContextUser.Email}}</a>
-					<a href="{{AppSubUrl}}/user/settings#privacy-user-settings">
-						{{if .ShowUserEmail}}
-							<i data-tooltip-content="{{ctx.Locale.Tr "user.email_visibility.limited"}}">
-								{{svg "octicon-unlock"}}
-							</i>
-						{{else}}
-							<i data-tooltip-content="{{ctx.Locale.Tr "user.email_visibility.private"}}">
-								{{svg "octicon-lock"}}
-							</i>
-						{{end}}
+					<a class="flex-text-inline" href="{{AppSubUrl}}/user/settings#privacy-user-settings" data-tooltip-content="{{ctx.Locale.Tr (Iif .ShowUserEmail "user.email_visibility.limited" "user.email_visibility.private")}}">
```

Comment:
Hmm, it looks like an abuse.

The `flex` and `align items` should be on the parent element `<li>` to align the `svg mail` / `mailto` / `svg private`.

No idea why it is only on the `svg private` .....

Reply:
![image](https://github.com/go-gitea/gitea/assets/3371163/731e8d32-aa6c-4b31-9d7d-de16b0b98949)
As in the screenshot, the `li` element has already been styled appropriately. The current Pull Request resolves the issue where the `svg` element inside the `a` element is not centered.

Reply:
Yes I agree `flex-text-inline` is likely not ideal, but it's aiming at this specific alignment problem that for some reason only shows on Windows, it does not reproduce on any MacOS browser for me.

Reply:
I guess the root problem is the "spaces" between elements, which make the height different.

I guess using `{{- svg ... -}}` would make the height correct and no need to use "flex" on "a". Just a guess and using "flex" is still acceptable.

Reply:
Yes, likely it is whitespace-related because the `display: flex` removes the whitespace rendering.